### DISCUSSION TITLE
Start Menu: Disable Web Search

### DISCUSSION THREAD

**Discussion by davidfregoli** [valinet/ExplorerPatcher#3223] [Ideas]
Would be nice to have a simple toggle to disable web search

> HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Search 
> 
> Make a new DWORD (32-bit) called: BingSearchEnabled
> 
> Ensure the value = 0

**Comment by dawidmosk** [comment]
Copy ready.
```coffeescript
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Search]
"BingSearchEnabled"=dword:00000000

[HKEY_CURRENT_USER\Software\Policies\Microsoft\Windows]
"DisableSearchBoxSuggestions"=dword:00000001

```

**Comment by rophel** [comment]
Agreed.  This is much needed as an option.
